From 345af3c33b64831f5421caf017906f74a9241d98 Mon Sep 17 00:00:00 2001
From: Felipe Rodrigues Monteiro Sousa <rms.felipe@gmail.com>
Date: Thu, 4 Oct 2012 17:40:31 -0400
Subject: [PATCH 3/4] updating in the bounds related to test case algorithm63

Signed-off-by: Felipe Rodrigues Monteiro Sousa <rms.felipe@gmail.com>
---
 .../esbmc-algorithm/algorithm63/test.desc          |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/regression/esbmc-cpp/esbmc-algorithm/algorithm63/test.desc b/regression/esbmc-cpp/esbmc-algorithm/algorithm63/test.desc
index 38ab439..87709b6 100644
--- a/regression/esbmc-cpp/esbmc-algorithm/algorithm63/test.desc
+++ b/regression/esbmc-cpp/esbmc-algorithm/algorithm63/test.desc
@@ -4,7 +4,7 @@
   <item_02_id>esbmc-algorithm</item_02_id>
   <item_03_summary>Aims to analyze the main function from module: esbmc-algorithm</item_03_summary>
   <item_04_file_C_to_test>main.cpp </item_04_file_C_to_test>
-  <item_05_option_to_run_esbmc>--unwind 10 --no-unwinding-assertions -I ~/libraries/</item_05_option_to_run_esbmc>
+  <item_05_option_to_run_esbmc>--unwind 4 --no-unwinding-assertions -I ~/libraries/</item_05_option_to_run_esbmc>
   <item_06_expected_result>^VERIFICATION SUCCESSFUL$</item_06_expected_result>
   <item_07_test_importance>High</item_07_test_importance>
   <item_08_execution_type>Automated</item_08_execution_type>
-- 
1.7.5.4

